tests/e2e/Services/Account/AccountBase.php

```
@@ -443,4 +443,47 @@ public function testAccountAbuseReset(): void
+        ]);
+
+        $this->assertEquals(201, $account['headers']['status-code']);
+
+        // Successful requests up to the limit should all pass
+        for ($i = 0; $i < 20; $i++) {
+            $session = $this->client->call(Client::METHOD_POST, '/account/sessions/email', $baseHeaders, [
+                'email' => $email,
+                'password' => $password,
+            ]);
+
+            $this->assertEquals(201, $session['headers']['status-code'], 'Request ' . ($i + 1) . ' of ' . $emailAbuseLimit . ' should succeed.');
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$emailAbuseLimit'. (undefined)

(UnusedLocalVariable)